app/static/src/router.ts

```diff
@@ -23,15 +26,21 @@ export function initialiseRouter(
     }
     const routeBase = `${basePath}/${appsPath}/${appName}`;
 
+    const appRoute: RouteRecordRaw = {
+        path: "/",
+        component: appComponent,
+        props: {
+            initSelectedTab: STATIC_BUILD ? "Options" : undefined
+        } as Parameters<NonNullable<(typeof appComponent)["setup"]>>["0"]
```

Comment:
hoorayyy to typescript, i can extract out the types of the arguments of `setup` function which are the props in position `0` which means we have sweet type checking on the tabs which were previously just strings, this isnt perfect since we could set it as "Data" but better than nothing!